js/targetCowFunctions.js

```diff
@@ -32,7 +32,7 @@ function lose_life() {
     var remaining_lives = parseInt($(".lives_value").text()) - 1
     $(".lives_value").text(remaining_lives)
 
-    if (remaining_lives === 0) {
+    if (remaining_lives < 1) {
```

Comment:
Seems fine but why? Wont make you change it just curious

Reply:
In the past we had an occasional problem with the number of lives going negative, at which point you can never die and just keep going more negative. We can't reproduce it, but in case we ever have a regression we put this in as a failsafe.